docs/upload-your-snap.md

```diff
@@ -1,69 +1,157 @@
-# Upload to the store
+---
+title: "Publish your snap"
+---
 
-So you've been working hard on your snap, and you finally have it to the point
-where you're ready to share it with the world? Great! You can use Snapcraft to
-upload it directly to the store.
 
-## The grade option
+In order to share your snaps with the world, you need to publish them in a
```

Comment:
Ok, but let's not forget that "publishing software" is the bread and butter of a lot of companies we are targeting with this content.